com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md

```diff
@@ -274,6 +279,20 @@ sealed class GrayScaleEditor : VolumeComponentEditor
 ```
 This custom editor is not really useful as it produces the same result as the editor that Unity creates. Custom Volume component editors also support an [additonal properties toggle](More-Options.md). To add it, you have to set hasAdvancedMode override to true. Then, inside the OnInspectorGUI, you can use the isInAdvancedMode boolean to display more properties.
 
+## Dynamic resolution and DLSS support
+
+If you are planning to fully take advantage of DLSS and / or dynamic resolution on your pass, and you require to interpolate / uv sample either color / normals or velocity, you must use the following functions to calculate the correct uvs:
```

Comment:
Phrasing change: 

If you want to use DLSS and/or dynamic resolution on your pass, and you need to interpolate or sample UVs from color / normal or velocity, you must use the following functions to calculate the correct UVs: